<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans SemiBold</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: Open Sans SemiBold;
	font-color: white;
}


QPushButton{
    background-color: #2e9cca;
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color: #aaabb8;
	color: white;
}

QPushButton:pressed {
	border-width: 5px;
	border-color: white;
	
}

QPushButton:hover{
	background-color: #2964BA;
}

QPushButton:disabled{
	background-color: #464866
}

#centralwidget{
	background: #2A1B3D;
}

QLineEdit{
	background-color: #aaabb8;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: white;
	selection-background-color: dark grey;
	color: black;
	padding: 2px;
    font: bold;
	font-size: 16px;
	alignment: centre;

}

QLabel{
	color: white;
	font: bold;

}

QTableWidget{
	background-color: #464866;
	color: white;
	selection-background-color: #2964BA;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 1px;
	border-radius: 15px;
	gridline-color: #aaabb8;
}

QRadioButton{
	color: white;
}

QComboBox{
	background: #464866;
	color: white;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
}

QComboBox:hover{
	background: #2964BA
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="q_next_question">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_leave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_previous_question">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QWidget" name="q_left_matrix" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>160</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="q5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>21</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="l3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>20</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_check_correct">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="q_quiz_name">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Quiz Name</string>
    </property>
   </widget>
   <widget class="QWidget" name="q_right_matrix" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="Line" name="line_18">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>20</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q15">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q17">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q14">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q18">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_14">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>21</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="q_question_number">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Question 1/9</string>
    </property>
   </widget>
   <widget class="QLabel" name="q_by_username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>By Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="q_operator_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 40px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="q_equals">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 40px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="q_answer_matrix" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="q23">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q20">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q25">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q21">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l2_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>21</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="l3_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l4_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l6_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l5_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>20</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l1_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q24">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q22">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q27">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="q_single_answer">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
